Python/bytecodes.c

```diff
@@ -1507,10 +1507,9 @@ dummy_func(
 
         op(_LOAD_GLOBAL, ( -- res, null if (oparg & 1))) {
             PyObject *name = GETITEM(FRAME_CO_NAMES, oparg>>1);
-            PyObject *res_o = _PyEval_LoadGlobal(GLOBALS(), BUILTINS(), name);
-            ERROR_IF(res_o == NULL, error);
+            _PyEval_LoadGlobalStackRef(GLOBALS(), BUILTINS(), name, &STACK_ENTRY(res));
```

Comment:
```suggestion
            _PyEval_LoadGlobalStackRef(GLOBALS(), BUILTINS(), name, &res[0]));
```

Can we go back to the earlier design of making `res` an array?
Please also add a comment explaining why we `res` needs to be trated specially (we need pass a pointer to `_PyEval_LoadGlobalStackRef`).

The code generator already understands arrays, and it avoids confusion as to where it should insert stack spills for fully deferred reference counting.

You'll able to remove `def stack_entry` below.

Sorry for the back and forth on this.